.travis.yml

```diff
@@ -19,9 +19,10 @@ env:
     - TARGETS="-C libbeat testsuite ES_HOST=elasticsearch-210"
     - TARGETS="-C topbeat testsuite"
     - TARGETS="-C filebeat testsuite"
+    - TARGETS="-C winlogbeat testsuite"
     - TARGETS="-C packetbeat testsuite"
     - TARGETS="-C libbeat crosscompile"
-    - TARGETS="-C filebeat crosscompile"
+    - TARGETS="-C winlogbeat crosscompile"
```

Comment:
It does. For example, you are building on your mac and you run `make crosscompile` to build the Windows executables (it only crosscompiles for `GOOS=windows`). It is consistent Makefile behavior as with the other beats.